pkg/gatherers/clusterconfig/sap_pods.go

```diff
@@ -16,23 +16,34 @@ import (
 	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherSAPPods collects information about pods running in SAP/SDI namespaces.
-// Only pods with a failing status are collected.
-// Failed pods belonging to a job that has later succeeded are ignored.
+// GatherSAPPods Collects information about pods running in SAP/SDI namespaces.
 //
-// **Conditional data**: This data is collected only if the "installers.datahub.sap.com" resource is found in the cluster.
+// - Only pods with a failing status are collected.
+// - Failed pods belonging to a job that has later succeeded are ignored.
 //
-// Relevant Kubernetes API docs:
-//   - https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/core/v1
-//   - https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/batch/v1
-//   - https://pkg.go.dev/k8s.io/client-go/dynamic
+// ### API Reference
+// - https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/core/v1
+// - https://pkg.go.dev/k8s.io/client-go/kubernetes/typed/batch/v1
+// - https://pkg.go.dev/k8s.io/client-go/dynamic
 //
-// * Location in archive: config/pod/{namespace}/{pod-name}.json
-// * Id in config: clusterconfig/sap_pods
-// * Since versions:
-//   - 4.6.24+
-//   - 4.7.5+
-//   - 4.8+
+// ### Sample data
+// None
+//
+// ### Location in archive
+// - `config/pod/{namespace}/{name}.json`
+//
+// ### Config ID
+// `clusterconfig/sap_pods`
+//
+// ### Released version
+// - 4.8.2
+//
+// ### Backported versions
+// - 4.7.5+
+// - 4.6.25+
```

Comment:
Is this wrong or was it originally wrong? There was `4.6.24+`

Reply:
Based on the information that I found, it was wrong. You can confirm: [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1935775) [RN](https://access.redhat.com/errata/RHBA-2021:1153)